Fix some typos in gradient code

hortont424 opened the pull request:
#### fca37e65b986d74e2e489869679a430ba5e71dd2
<pre>
Fix some typos in gradient code
<a href="https://bugs.webkit.org/show_bug.cgi?id=242936">https://bugs.webkit.org/show_bug.cgi?id=242936</a>

Reviewed by Simon Fraser and Sam Weinig.

* Source/WebCore/platform/graphics/cg/GradientRendererCG.cpp:
(WebCore::analyzeColorStopsForEmulatedAlphaPremultiplicationOpportunity):
(WebCore::alphaTransformStopsToEmulateAlphaPremultiplication):
(WebCore::GradientRendererCG::pickStrategy const):
(WebCore::analyzeColorStopsForEmulatedAlphaPremuliplicationOppertunity): Deleted.
(WebCore::alphaTransformStopsToEmulateAlphaPremuliplication): Deleted.
Premuliplication-&gt;Premultiplication, Oppertunity-&gt;Opportunity.

Canonical link: <a href="https://commits.webkit.org/252833@main">https://commits.webkit.org/252833@main</a>
</pre>

weinig reviewed (approved):
Oh so what, now I need to know how to spell!

hortont424 committed:
Fix some typos in gradient code
https://bugs.webkit.org/show_bug.cgi?id=242936

Reviewed by Simon Fraser and Sam Weinig.

* Source/WebCore/platform/graphics/cg/GradientRendererCG.cpp:
(WebCore::analyzeColorStopsForEmulatedAlphaPremultiplicationOpportunity):
(WebCore::alphaTransformStopsToEmulateAlphaPremultiplication):
(WebCore::GradientRendererCG::pickStrategy const):
(WebCore::analyzeColorStopsForEmulatedAlphaPremuliplicationOppertunity): Deleted.
(WebCore::alphaTransformStopsToEmulateAlphaPremuliplication): Deleted.
Premuliplication->Premultiplication, Oppertunity->Opportunity.

Canonical link: https://commits.webkit.org/252833@main
